app-admin/cdist/cdist-5.1.3.ebuild

```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python{3_5,3_6,3_7} )
+
+inherit distutils-r1
+
+DESCRIPTION="A usable configuration management system"
+HOMEPAGE="https://www.nico.schottelius.org/software/cdist/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
```

Comment:
I see upstream has some sort of tests included at least in their code repo, 
https://code.ungleich.ch/ungleich-public/cdist/tree/master/cdist/test
if they are available in the pypi tarball could you look into implementing them in the ebuild?

Reply:
This test are unittest to test the python code (cdist core) after changes, we run the tests bevor we release to pypi. In consequence, the test is NOT included in the release tarbal. We have only python as dependency for the core, so there is also no benefit to run the tests again on each and every system.

Reply:
Right, thanks!